apps/omg_utils/lib/omg_utils/http_rpc/client.ex

```diff
@@ -32,23 +32,28 @@ defmodule OMG.Watcher.HttpRPC.Client do
   Gets Block of given hash
   """
   @spec get_block(binary(), binary()) :: response_t()
-  def get_block(hash, url) do
-    %{hash: Encoding.to_hex(hash)}
-    |> Adapter.rpc_post("block.get", url)
-    |> Adapter.get_response_body()
-    |> decode_response()
-  end
+  def get_block(hash, url), do: call(%{hash: Encoding.to_hex(hash)}, "block.get", url)
+
+  def get_exit_data(encoded_position, url), do: call(%{utxo_pos: encoded_position}, "utxo.get_exit_data", url)
```

Comment:
these additions don't fit here :(. See the moduledoc for this. 

Secondly, I think moving this to the generic Utils module is also debatable (e.g. you're forcing the non-test modules to have access to Watcher's RPC too, e.g. `block_getter.ex`, which makes no sense).

Lastly, I think we have some `support/test_helper.ex` modules to talk to the Watcher RPC, can't we use that?

Reply:
Well in my opinion they fix in `HttpRpc.Client` the most. My reasoning is that it's a single rpc client that understands responses from both ch-ch and Watcher. 

Also please note that Watcher TestHelpers we use to test Watcher's controllers does not use http, shortcut the communication through Phoenix test features so nothing even listen on the network.

From `ByzantineEvents` we have no luck and we need to make a full http call.

> you're forcing the non-test modules to have access to Watcher's RPC too

Well `HttpRpc.Client` DRYs things a lot regarding making call and parse the response. Also providing just ch-ch's url the additional functions wouldn't work.

Reply:
I reverted the changes that moves HttpRPC.Client to OMG.Utils and Watcher RPC calls are made only from ByzantineEvents module.